gensrc/thrift/FrontendService.thrift

```diff
@@ -74,6 +74,7 @@ struct TDescribeTableParams {
   3: optional string user   // deprecated
   4: optional string user_ip    // deprecated
   5: optional Types.TUserIdentity current_user_ident // to replace the user and user ip
+  6: optional i32 limit
```

Comment:
`limit` should not be add in TDescribeTableParams
I think TPlanNode contains the limit, you can refer to be/src/exec/vectorized/olap_scan_node.cpp to see how to get the limit.

Reply:
`_limit` is the ExecNode field. so you could directly access limit in SchemaScanNode.

please refer to 

```
    _num_rows_returned += chunk_dst->num_rows();
    if (reached_limit()) {
        int64_t num_rows_over = _num_rows_returned - _limit;
        chunk_dst->set_num_rows(chunk_dst->num_rows() - num_rows_over);
        COUNTER_SET(_rows_returned_counter, _limit);
    } else {
        COUNTER_SET(_rows_returned_counter, _num_rows_returned);
    }
    *chunk = std::move(chunk_dst);
```

Reply:
The limit parameter is passed from BE to FE, then which can reduce the amount of data transmitted from FE to BE.